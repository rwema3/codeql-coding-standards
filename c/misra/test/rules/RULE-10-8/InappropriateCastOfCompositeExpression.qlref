rules/RULE-10-8/InappropriateCastOfCompositeExpression.ql