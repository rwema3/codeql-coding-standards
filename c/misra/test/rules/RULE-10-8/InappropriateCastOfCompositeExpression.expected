| test.c:4:16:4:20 | ... + ... | Cast from essentially Unsigned type to essentially Signed type changes type category. |
| test.c:5:18:5:22 | ... + ... | Cast from essentially Signed type to essentially Unsigned type changes type category. |
| test.c:14:18:14:24 | ... + ... | Cast from essentially Unsigned type to essentially Unsigned type widens type. |
| test.c:20:16:20:22 | ... + ... | Cast from essentially Signed type to essentially Signed type widens type. |
